src/NUnitFramework/nunitlite.tests/TextUITests.cs

```diff
@@ -527,18 +527,17 @@ public void TwoTests_NestedExecution(string labels, string expected)
             var textUI = CreateTextUI(labels);
 
             textUI.TestStarted(test1);
-            textUI.TestOutput(new TestOutput("Immediate output from first test\n", "", test1.FullName));
+            textUI.TestOutput(new TestOutput("Immediate output from first test\n", "", null, test1.FullName));
             textUI.TestStarted(test2);
-            textUI.TestOutput(new TestOutput("Another immediate output from first test\n", "", test1.FullName));
-            textUI.TestOutput(new TestOutput("Immediate output from second test\n", "", test2.FullName));
+            textUI.TestOutput(new TestOutput("Another immediate output from first test\n", "", null, test1.FullName));
+            textUI.TestOutput(new TestOutput("Immediate output from second test\n", "", null, test2.FullName));
             result2.OutWriter.WriteLine("OUTPUT2");
             textUI.TestFinished(result2);
             result1.OutWriter.WriteLine("OUTPUT1");
             textUI.TestFinished(result1);
 
             Assert.That(Report, Is.EqualTo(expected));
         }
-
```

Comment:
This isn't a big deal, but if you see accidental changes like this can you undo them them?

Reply:
@jnm2  I'm having hard time with line feed. Giving that the change was made in previous commit, How can I undo it in later commit?

Reply:
I see you added a new line to reverse what you did which is great. What you're seeing now is that instead of adding an empty line, you added a line with eight spaces in it. If you want to get rid of this change, just delete the eight spaces and there will no longer be a net change in this PR. But feel free to leave this if you've already put enough time into it; it's not a big deal.

Reply:
Let me give it another try